apps/browser/qml/pages/components/BrowserListItem.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+ListItem {
+    id: root
+
+    property alias label: valueButton.label
+    property alias value: valueButton.value
+    property alias iconSource: icon.source
+    property alias secondaryText: secondaryLabel.text
+
+    width: parent.width
+    contentHeight: Math.max(Theme.itemSizeMedium, row.height)
+    onClicked: openMenu()
+
+    Row {
+        id: row
+
+        width: parent.width - 2*Theme.horizontalPageMargin
+        x: Theme.horizontalPageMargin
+        anchors.verticalCenter: parent.verticalCenter
+
+        Icon {
+            id: icon
+
+            anchors.verticalCenter: parent.verticalCenter
+        }
+
+        Column {
+            width: parent.width
+            anchors.verticalCenter: parent.verticalCenter
+
+            ValueButton {
+                id: valueButton
+
+                width: parent.width - icon.width
+                labelColor: root.highlighted ? palette.highlightColor : palette.primaryColor
+                valueColor: Theme.highlightColor
+                opacity: 1.0
+                enabled: false
+            }
+            Label {
+                id: secondaryLabel
```

Comment:
Rather than creating a separate `Label` here, there is the `description` text of the `ValueButton`. It looks like this can be used instead.